From: Didier Raboud <odyx@debian.org>
Date: Sun, 4 Oct 2020 16:29:03 +0200
Subject: 3.20.6+ regression: In models.dat,
 take 3.20.5's "plugin=1" for all models with 1 in least significant bit of
 plugin-reason (="needs plugin for printing")

---
 data/models/models.dat | 68 +++++++++++++++++++++++++-------------------------
 1 file changed, 34 insertions(+), 34 deletions(-)

diff --git a/data/models/models.dat b/data/models/models.dat
index 3f185f7..dc6ba76 100644
--- a/data/models/models.dat
+++ b/data/models/models.dat
@@ -17710,8 +17710,8 @@ model1=HP LaserJet P1009 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
-plugin-reason=0
+plugin=1
+plugin-reason=1
 power-settings=0
 pq-diag-type=0
 r-type=0
@@ -18353,7 +18353,7 @@ model1=HP LaserJet Pro CP1025 Color Printer Series
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -18989,7 +18989,7 @@ model1=HP LaserJet Professional P 1102w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19032,7 +19032,7 @@ model1=HP Laserjet Professional P1106w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19075,7 +19075,7 @@ model1=HP Laserjet Professional P1106 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19118,7 +19118,7 @@ model1=HP LaserJet Professional P1107 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19161,7 +19161,7 @@ model1=HP LaserJet Professional P1107w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19204,7 +19204,7 @@ model1=HP Laserjet Professional P1108w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19247,7 +19247,7 @@ model1=HP Laserjet Professional P1108 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19290,7 +19290,7 @@ model1=HP LaserJet Professional P1109w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19333,7 +19333,7 @@ model1=HP LaserJet Professional P1109 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19683,7 +19683,7 @@ model2=HP LaserJet Professional M1132s Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19726,7 +19726,7 @@ model1=HP Laserjet Professional M1136 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19769,7 +19769,7 @@ model1=HP LaserJet Professional M1137 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19812,7 +19812,7 @@ model1=HP LaserJet Professional M1138 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19855,7 +19855,7 @@ model1=HP LaserJet Professional M1139 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20447,7 +20447,7 @@ model1=HP LaserJet Professional M1212nf Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20490,7 +20490,7 @@ model1=HP LaserJet Professional M1213nf Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20533,7 +20533,7 @@ model1=HP LaserJet Professional M1214nfh Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20675,7 +20675,7 @@ model1=HP LaserJet Professional M1216nfh MFP
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20718,7 +20718,7 @@ model1=HP Color LaserJet CP1217 Printer
 monitor-type=0
 panel-check-type=1
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -20813,7 +20813,7 @@ model1=HP Laserjet M1210 MFP Series
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20903,7 +20903,7 @@ model1=HP HotSpot LaserJet Pro M1218nfs MFP
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20945,7 +20945,7 @@ linefeed-cal-type=0
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20987,7 +20987,7 @@ linefeed-cal-type=0
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -21030,7 +21030,7 @@ model1=HP LaserJet Professional M1219nf MFP
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -24109,7 +24109,7 @@ model1=HP LaserJet Professional P1567
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24152,7 +24152,7 @@ model1=HP LaserJet Professional P1568
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24195,7 +24195,7 @@ model1=HP LaserJet Professional P1569
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24631,7 +24631,7 @@ model1=HP LaserJet Professional P1606dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24674,7 +24674,7 @@ model1=HP LaserJet Professional P1607dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24717,7 +24717,7 @@ model1=HP LaserJet Professional P1608dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24760,7 +24760,7 @@ model1=HP LaserJet Professional P1609dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
